packages/apptive_grid_theme/lib/src/apptive_grid_icons_icons.dart

```diff
@@ -7,23 +7,23 @@ abstract class ApptiveGridIcons {
   ApptiveGridIcons._();
 
   /// A More Icon
-  static const IconData more = MdiIcons.dotsHorizontalCircle;
+  static final IconData more = MdiIcons.dotsHorizontalCircle;
 
   /// Icon for Grids
-  static const IconData grid = MdiIcons.table;
+  static final IconData grid = MdiIcons.table;
 
   /// Icon for Forms
-  static const IconData form = MdiIcons.formTextbox;
+  static final IconData form = MdiIcons.formTextbox;
 
   /// Icon for Kanban Boards
-  static const IconData kanban = MdiIcons.trello;
+  static final IconData kanban = MdiIcons.trello;
 
   /// Icon for Calendar
-  static const IconData calendar = MdiIcons.calendar;
+  static final IconData calendar = MdiIcons.calendar;
 
   /// Icon for Map Views
-  static const IconData map = MdiIcons.googleMaps;
+  static final IconData map = MdiIcons.googleMaps;
 
   /// Icon for Gallery Views
-  static const IconData gallery = MdiIcons.viewGrid;
+  static final IconData gallery = MdiIcons.viewGrid;
```

Comment:
Was there a reaso/issue in the changelog of the Material Design Icons package explaining why these are not `const` anymore. A change like this does not allow to properly tree shake the icon fonts. So possibly we should avoid changing this (and the version as we might not need the new major version?)

Reply:
https://pub.dev/packages/material_design_icons_flutter/changelog
7.0.7296:
- Breaking Change: MdiIcons.* variables are no longer const, to avoid breaking tree-shaking. See [issue #54](https://github.com/ziofat/material_design_icons_flutter/issues/54)
- Support for Dart 3.0

Reply:
This seems to be related to fixing tree-shaking